research/src/pages/mission.research.mdx

```diff
@@ -32,6 +32,24 @@ These are the goals Open UI believes will realize the above vision.
 - Eventual browser standards for web app components
 - Converging designer processes and developer workflows
 
+## Goals for web specifications
+
+For web app component specifications here are the specific goals:
+
+- Standardize on the names of the component parts and anatomy structure
+- Standardize the name of states which are applicable to the component and its parts
+- Standardize the behaviors of components
+- Standardize the base style upon which authors can build a custom appearance
```

Comment:
Can you give an example of "base style". I'm not sure how to draw the line between that and the non-goal of non-standardization of "default appearance".

Reply:
Good question, the base style are things that may/may not be necessary for this group but whether or not the part should be BFCs, Inlines, etc. From a platform perspective we would want to align on this and then probably have a discussion regarding how to negate all of these and set a base set of styles (such as appearance: none does not necessarily set inlines to block, etc. It removes a certain set of styles to ensure that the component still functions). @BoCupp-Microsoft do you have anything to add to this?

Reply:
Bo and I discussed this a bit and this is definitely a platform requirement but not sure it needs to be in there for frameworks for now I'm going to remove it.